### PR TITLE
Only publish docs for PRs originating from a repo owned by Lime

### PR DESCRIPTION
**Pull request by adrianschmidt**
fix: Lundalogik/crm-feature#3014

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

**Review by anderssonjohan — APPROVED** [review]

**Comment by adrianschmidt** [comment]
Approved by @anderssonjohan, and I'm a code owner for this code myself, so together, I think we can merge without approval my another code owner 😛

**Pull request merged**
